statvar_imports/us_college_ipeds/preprocess.py

```diff
@@ -0,0 +1,36 @@
+import os
+import pandas as pd
```

Comment:
I don't see any usage of pandas in this file. Can we remove it from the list of imports then?